package.json

```diff
@@ -3,107 +3,56 @@
   "version": "32.0.1",
   "private": true,
   "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "lint:style": "d2-style js check --all",
+    "eject": "react-scripts eject",
+    "release": "packages release",
     "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
     "localize": "yarn extract-pot && d2-i18n-generate -n usage-analytics-app -p ./i18n/ -o ./src/locales/",
     "manifest": "d2-manifest manifest.json ./public/manifest.webapp",
-    "format": "packages code-style",
-    "release": "packages release",
     "prestart": "yarn localize && yarn manifest",
-    "start": "node scripts/start.js",
-    "prebuild": "rm -rf build && mkdir build && yarn format && yarn manifest && yarn localize",
-    "build": "node scripts/build.js",
-    "build-local": "yarn prebuild && yarn build",
-    "test": "node scripts/test.js",
-    "coverage": "npm test -- --coverage"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "packages commit-style",
-      "pre-commit": "packages code-style && yarn extract-pot && git add ./i18n/"
-    }
-  },
-  "babel": {
-    "presets": [
-      "react-app",
-      "@babel/preset-env"
-    ]
+    "prebuild": "yarn localize && yarn manifest",
```

Comment:
Do we need the manifest?

Reply:
Not sure actually. I'll see if I can find out

Reply:
Removed it (and related code), and locally nothing breaks. But I'm not sure if they might break anything in production. The app should be using the env vars for discovering the api though, which is the only thing I think it could break. So I think it should be fine. You?

Reply:
Could just convert it to a static manifest.json, without the dhis2 specific values

Reply:
Moved to a static manifest: https://github.com/dhis2/usage-analytics-app/pull/41/commits/ac43feecd97f1753cc8b08bdf02775ed9c256898. If you're allright with that this is ready to merge.